crates/pop-cli/src/commands/build/mod.rs

```diff
@@ -37,7 +37,7 @@ pub(crate) struct BuildArgs {
 	pub(crate) profile: Option<Profile>,
 }
 
-/// Build a parachain, smart contract, chain specification or Rust package.
+/// Build a chain specification and its genesis artifacts.
```

Comment:
So this is technically an umbrella type for commands, now build spec being the only one.

"Subcommand for building chain artifacts" might be an oke description to go with for now

Reply:
Is tricky being the only one subcommand, that's why I used the same comment. But probably you are right and is better to indicate is a subcommand in the command.

Reply:
https://github.com/r0gue-io/pop-cli/pull/394/commits/c3a724e9482308f8a4204aadcc5e433390b78f66

Reply:
Thank you!

Yeah otherwise the umbrella type should be removed but it might become useful in the future so think it is better to leave it in. It is also out of scope for now but keeping things correct and of a certain quality should always be strived for :)